src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerDriver.cs

```diff
@@ -2487,6 +2609,27 @@ void executeCodeBlockActions(ImmutableArray<SyntaxNode> executableCodeBlocks, IE
                     }
                 }
             }
+
+            static IEnumerable<ExecutableCodeBlockAnalyzerActions> getExecutableCodeBlockAnalyzerActions(
```

Comment:
```suggestion
            static IEnumerable<ExecutableCodeBlockAnalyzerActions> GetExecutableCodeBlockAnalyzerActions(
```

also, very wary about yield-return.

Reply:
Simplified this code to use an array builder instead of returning enumerable.